documentation/service-rules/ecf1/participants.md

```diff
@@ -0,0 +1,567 @@
+---
+title: Lead provider API
```

Comment:
could we make this accessible in https://ecf-service-manual.education.gov.uk/?

Reply:
Hi @cwrw It's accessible from https://ecf-service-manual.education.gov.uk/service-rules/ecf1/lead-provider-api/ ... as it's part of API documentation.. do you want me to add a link to it in the main page? in a new section for the API?